extensions/wikia/Blogs/BlogTemplate.php

```diff
@@ -265,7 +266,7 @@ class BlogTemplateClass {
         '/(<table[^>]*>|<\/table>)/i',
         '/(<tr[^>]*>|<\/tr>)/i',
         '/<td[^>]*>(.*?)<\/td>/i',
-        '/<th[^>]*>(.*?)<\/th>/ie',
+        '/<th[^>]*>(.*?)<\/th>/i',
```

Comment:
The replacement wasn't actually doing anything that required the ```/e``` flag.  Since the flag is deprecated (and throwing errors in dev) removed it.